src/dashboard/cd-dashboard-children.vue

```diff
@@ -14,6 +17,13 @@
         <p class="cd-dashboard-children__badges-none" v-else>{{ $t('{name} doesn\'t have any badges yet. Talk to the organisers of your Dojo to learn how {name} can be rewarded through badges.', { name: child.firstName }) }}</p>
       </div>
     </div>
+    <div v-else class="cd-dashboard-children">
```

Comment:
Seems like there's a slight issue with this PR. Not quite sure why, but these changes shouldn't be here since they're already in master... 😅

Reply:
Yeah I will rebase off master and fix dat

Reply:
Ok i did not fix that

Reply:
Do I have to force push? @DanielBrierton